sdk/resourcemanagerhybrid/README.md

```diff
@@ -131,7 +129,52 @@ AzureResourceManager azure = AzureResourceManager
     .withDefaultSubscription();
 ```
 
-The sample code assumes global Azure. Please change `AzureEnvironment.AZURE` variable if otherwise.
+The sample code assumes global Azure. To authenticate against Azure Stack Hub, create a custom azure environment that
+fetches information from your Azure Stack Hub's Azure Resource Manager metadata endpoint:
+
+```java
+private static AzureEnvironment getAzureEnvironmentFromArmEndpoint(String armEndpoint) {
```

Comment:
You might want to check this tool. https://github.com/Azure/azure-sdk-tools/blob/main/packages/java-packages/codesnippet-maven-plugin/README.md#injecting-codesnippets-into-readmes

It might not able to let you use a single Java for all these same examples in READMEs (I think it requires the Java at same package as the README), but it would at least make coding easier and avoid potential misspelling or syntax bug.

Reply:
Went and ahead and added the `BEGIN`/`END` sections the tool uses, and so changed all the line references to use the snippet identifiers instead.